desktop-widgets/diveplanner.cpp

```diff
@@ -169,9 +169,11 @@ DivePlannerWidget::DivePlannerWidget(QWidget *parent, Qt::WindowFlags f) : QWidg
 	connect(closeKey, &QShortcut::activated, plannerModel, &DivePlannerPointsModel::cancelPlan);
 
 	// This makes shure the spinbox gets a setMinimum(0) on it so we can't have negative time or depth.
-	ui.tableWidget->view()->setItemDelegateForColumn(DivePlannerPointsModel::DEPTH, new SpinBoxDelegate(0, INT_MAX, 1, this));
+	// Limit segments to a depth of 3000 m/ft and a duration of 100 h. The DivePlannerPointsModel will
+	// sanitize depth according to the current units.
+	ui.tableWidget->view()->setItemDelegateForColumn(DivePlannerPointsModel::DEPTH, new SpinBoxDelegate(0, 3000, 1, this));
```

Comment:
I know this is ridiculous nit picking... but... we should do 1000m/3000ft or 3000m/9000ft - having a different max depth depending on metric or imperial just... bugs me.

Reply:
Sure, we can catch it here, but then the "item delegate" would have to be changed when the user changes the units. Probably no big deal - I will see what I can do.

Reply:
Done.